game/scripts/npc/abilities/luna_moon_glaive.txt

```diff
@@ -24,7 +24,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bounces"                                         "1 2 3 4 5 6" //OAA
+        "bounces"                                         "3 4 5 6 7 8" //OAA
```

Comment:
lmao this is pushing it too far.

Reply:
Every Moon Glaive bounce can crit, remember?

Reply:
Ive decreased the bounces by 1 at each level. It should start off higher then 1 bounce at level 1 though. This heroes farming speed is bad and I don't think moon glaives is a crazy carry ability vs a full team of 6 slotted heroes (crimson guard hard counters it).